configs/oasis-borrow/index.ts

```diff
@@ -42,15 +42,139 @@ export default function ({
     },
     parameters: {
       topBanner: {
-        name: "rebranding",
-        url: "https://blog.summer.fi/oasis-app-rebrands-to-summer-fi/",
-        message: "Oasis.app is now Summer.fi! Read the announcement",
+        name: 'rebranding',
+        url: 'https://blog.summer.fi/oasis-app-rebrands-to-summer-fi/',
+        message: 'Oasis.app is now Summer.fi! Read the announcement',
       },
       aaveLike: {
         orderInformation: {
           showFlashloanInformation: notProduction,
         },
       },
+      navigation: {
```

Comment:
Two suggestions that will improve readability of this file.

1. How about moving `navigation` one level up, so it's on the same level as `features` and `parameters`?
2. I was thinking about creating three separated files, `features.ts`, `parameters.ts` and `navigation.ts` and importing them to the index? Then, in index file it would be just
```
return { features, navigation, parameters }
```

Reply:
These would need to be functions as we pass some helper parameters.